src/beman/optional26/tests/optional.t.cpp

```diff
@@ -868,3 +868,42 @@ TEST(OptionalTest, HashTest) {
         EXPECT_EQ(h1, h2);
     }
 }
+
+// Moving an `optional<T&>` should not move the remote value.
+// Check this by deleting the rvalue overloads of some (unrealistic) types.
+TEST(OptionalTest, MoveRef) {
+    struct copyable_but_not_movable {
```

Comment:
same obersation - we have a header file with test classes in src/beman/optional26/tests/test_types.hpp

Reply:
Thanks for the pointer! I'll update the PRs accordingly.